lib/main_common.pm

```diff
@@ -2658,4 +2661,25 @@ sub load_lvm_tests {
     }
 }
 
+sub load_mm_autofs_tests {
```

Comment:
You can just use the "extra test" model here as well. This way you do not need to define anything in `products/opensuse/main.pm` nor `products/sle/main.pm`. Just name the method `load_extra_tests_mm_autofs` and define the EXTRATEST=mm_autofs variable